src/index.ts

```diff
@@ -96,7 +96,17 @@ class SunCard extends LitElement {
     const [time, period] = result
     return { time, period }
   }
-
+  calculateTimeBetweenDates(date1: Date, date2: Date): TSunCardTime {
+    // Returning time format is "HH:mm"
+    const minutes = Math.abs(
+      this.convertDateToMinutesSinceDayStarted(date1) -
+        this.convertDateToMinutesSinceDayStarted(date2)
+    )
+    const time = `${Math.floor(minutes / 60)}:${("00" + (minutes % 60)).slice(
+      -2
+    )}`
+    return { time }
```

Comment:
This could be simplified greatly I think:
```js
const timeMs = date2 - date1
if (timeMs < 0) {
  return { time: '0:00' }
else {
  const time = new Date(timeMs)
  return { time: `${time.getUTCHours()}:${time.getUTCMinutes()}` }
}
```

Reply:
I think the biggest issue we should address here is the format, I learn recently (https://github.com/AitorDB/home-assistant-sun-card/pull/29) that not all locales uses `:` so separate hours and minutes. Also some of them will use 2 digits always (08:00 vs 8:00) so I would suggest to add also `toLocaleTimeString` similar to `parseTime`